scripts/install.sh

```diff
@@ -70,37 +70,31 @@ fi
 
 KUBEBUILDER_VERSION=${KUBEBUILDER_VERSION#"v"}
 KUBEBUILDER_VERSION_NAME="kubebuilder_${KUBEBUILDER_VERSION}"
-KUBEBUILDER_DIR=/usr/local/kubebuilder
+KUBEBUILDER_DIR=/usr/local/bin/kubebuilder
 
```

Comment:
I think we need to check where this script has been used. 
Did you test these changes? Could you please add in the first comment how are you testing the changes and its result?